lib/vector.dart

```diff
@@ -0,0 +1,88 @@
+library flame.vector_math;
+
+export 'package:vector_math/vector_math_64.dart' show Vector2;
+
+import 'dart:math';
+import 'dart:ui';
+import 'package:vector_math/vector_math_64.dart';
+
+class VectorUtil {
```

Comment:
Couldn't we use extensions to extend Vector2?

Reply:
I wish, but since the it uses factories in the constructors like this:
`factory Vector2(double x, double y) => Vector2.zero()..setValues(x, y);`
I haven't found a way to extend it nicely.
If you have any suggestions of how to do it I'm all ears.

Reply:
I see, fair enough.

I particularly, don't like the `Utils` name, I would suggest naming this `VectorOperations` or something like.

Either way, this is just a personal preferences, and a small one, not a blocker for this PR.

Reply:
Good idea, maybe `Vector2Operations`? Since it doesn't apply for the other vector types in `vector_math`

Reply:
SGTM!